airflow/providers/papermill/provider.yaml

```diff
@@ -57,6 +57,7 @@ dependencies:
   - ipykernel
   - pandas>=2.1.2,<2.2;python_version>="3.9"
   - pandas>=1.5.3,<2.2;python_version<"3.9"
+  - python3-saml>=1.16.0
```

Comment:
In the description you mentioned its being used inly by Amazon provider. If so why doo we need it in papermill?

Reply:
Only in Amazon in Papermill - that was typed to fast when I was in a hurry for Airflow Summit :)

Reply:
what a mistake. Thanks for noticing it :)